kusama-guide/docusaurus.config.js

```diff
@@ -171,7 +171,7 @@ module.exports = {
     announcementBar: {
       id: "banner",
       content:
-        'Join Polkadot Decoded 2023 on June 28-29! <a href="https://polkadot.network/ecosystem/events/decoded-2023/?utm_source=guide.kusama.network&utm_medium=referral&utm_campaign=decoded23&utm_content=notification&utm_term=two" target="_blank" rel="noopener noreferrer">Register now 👉</a>',
+        'sub0 is back! Lisbon, Portugal, Sep 19-20. <a href="https://polkadot.network/ecosystem/events/sub0/?utm_source=guide.kusama.network&utm_medium=referral&utm_campaign=sub023&utm_content=notification" target="_blank" rel="noopener noreferrer">Learn more 👉</a>',
```

Comment:
@DrW3RK I think sub0 is lowercase, at least on the official website everything is lowercase. 

My other question is: **Why not promoting ParisDOT.comm that comes first?**